.github/workflows/build-website.yml

```diff
@@ -21,7 +21,9 @@ jobs:
 
       - run: ./src/scripts/structure_mastg.sh
 
-      - run: python3 src/scripts/transform_files.py
+      - env:
```

Comment:
It might work without this. The GITHUB_TOKEN is created automatically on each run and should be accessible to the script via env.

Reply:
Let's try. We can always revert the commit that removes this declaration

Reply:
fa400261a285969b7777d533403d34db6425b17d